web/src/server_events_dispatch.js

```diff
@@ -246,6 +249,12 @@ export function dispatch_normal_event(event) {
                             window.electron_bridge.send_event("realm_name", event.value);
                         }
 
+                        if (event.property === "invite_to_realm_policy") {
+                            settings_invites.update_invite_user_panel();
+                            ui_init.update_invite_user_option();
```

Comment:
I'm pretty sure it's going to be an import cycle to have `server_events_disptach` call a function in the `ui_init` module. I think what we should do is move `update_invite_user_option` into either `gear_menu.js` to or maybe better a new `right_sidebar_ui.js` module that's responsible for that little component.

This is OK to merge since we're not trying to avoid import cycles in this module right now, but please start a #frontend discussion about this detail.